### PR TITLE
Added responsive sidebar

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Added responsive sidebar to home page
Screenshot (37)

### CODE REVIEW

**Review by poojakelkar — COMMENTED** [review]
Check the radio button It's not working, Otherwise looks good to merge. Nice design

**Review by tushargoyal812 — COMMENTED** [review]
i think no changes required,good to merge.

**Comment by SHIKHASINGH1506** [comment]
Merging this branch to dev, as no changes are required.

File: Components/Navbar/navbar.css [hunk 1]
```diff
@@ -101,63 +102,53 @@
...
   width: 100vw;
   z-index: 3;
   visibility: hidden;
-  opacity: 0;
   transition: all 0.3s cubic-bezier(0.25, 0.46, 0.43, 0.96);
 }
-/* .sidebar-wrapper,
 .sidebar {
+  padding: 2rem 1rem 1rem;
+  max-width: 16.875rem;
```
Comment (viru2001) [inline Components/Navbar/navbar.css:127]: is your root font-size 16px? that's why this number 16.875?

File: Components/Navbar/navbar.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+navbarToggler = document.querySelector("#sidebar-toggler");
+sidebarWrapper = document.querySelector("#sidebar-wrapper");
+sidebar = document.querySelector("#sidebar");
+
+navbarToggler.addEventListener('click', () => {
+    sidebarWrapperClasses = sidebarWrapper.classList;
+    sidebarClasses = sidebar.classList;
+    sidebarWrapperClasses.toggle("show");
+    sidebarClasses.toggle("view");
+});
```
Comment (viru2001) [inline Components/Navbar/navbar.js:10]: you can add a comment here for what is this code for.

**Review by viru2001 — COMMENTED** [review]
LGTM ! just minor suggestions

**Pull request merged**